Makefile

```diff
@@ -28,11 +28,13 @@ all-push: all-push-images push-manifest
 
 k8s-tars/${K8S_VERSION}/${ARCH}/${K8S_SERVER_TARBALL}:
 	mkdir -p k8s-tars/${K8S_VERSION}/${ARCH}
-	cd k8s-tars/${K8S_VERSION}/${ARCH} && curl -sSLO --retry 5 https://dl.k8s.io/${K8S_VERSION}/${K8S_SERVER_TARBALL}
-
+	cd k8s-tars/${K8S_VERSION}/${ARCH} && curl -sSLO --retry 5 https://dl.k8s.io/${K8S_VERSION}/${K8S_SERVER_TARBALL} \
+	&& curl -sSLO --retry 5 https://github.com/rancher/kubernetes/releases/download/${K8S_VERSION}-rc.1-rancher1/kubelet-${K8S_VERSION}-${ARCH}.tar.gz
```

Comment:
this is temporary right...?

Reply:
yep, final release tag will be only `${K8S_VERSION}`